src/ucp/core/ucp_context.c

```diff
@@ -1786,35 +1789,35 @@ ucp_memory_type_detect_mds(ucp_context_h context, const void *address, size_t si
     return UCS_MEMORY_TYPE_HOST;
 }
 
-uint64_t ucp_context_dev_tl_bitmap(ucp_context_h context, const char *dev_name)
+ucp_tl_bitmap_t ucp_context_dev_tl_bitmap(ucp_context_h context, const char *dev_name)
```

Comment:
bad idea to return structure - it could not be passed as register value

Reply:
Would it be better to change the function, so that it gets a pointer to a struct and fills it?
We need to pass this struct in many places, and upon discussion with @yosefe he told it's ok to pass it by value.